geo/src/algorithm/contains/triangle.rs

```diff
@@ -11,9 +11,33 @@ where
     T: GeoNum,
 {
     fn contains(&self, coord: &Coord<T>) -> bool {
-        let ls = LineString::new(vec![self.0, self.1, self.2, self.0]);
-        use crate::utils::{coord_pos_relative_to_ring, CoordPos};
-        coord_pos_relative_to_ring(*coord, &ls) == CoordPos::Inside
+        // leverageing robust predicates
```

Comment:
Let's go with this impl. if it's correct, rather than the non-robust one.

For quick intro. to robust, you may want to refer [this example](https://github.com/georust/robust/tree/main/examples/predicate-map) that explains why it's important.